### DISCUSSION TITLE
The ability to individually limit the rate at which users in a role can request an interface

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#14144] [Feature Requests]
For example, admin limit of 50 requests a second, admin2 limit of 100 requests a second

**Comment by rijkvanzanten** [comment]
This is a very similar discussion as https://github.com/directus/directus/pull/12736. The main problem is that we need to rate-limit _before_ the user is authenticated and thus the role is known, otherwise you can DDoS the authentication service 🤔 In order to support a per-user rate limit, we'd have to have a global rate limit first, and then a secondary rate limit that is controlled by the user
